index.js

```diff
@@ -1,10 +1,16 @@
 const postcss = require('postcss')
 const matchImports = /^((?:.+?|\([\s\S]+?\))\s+from\s+)(['"])(@?\w+.*?)\2$/
+const matchImportPath = /^(['"])(@?\w.*)/
```

Comment:
Can we use we the same regex as in `matchImports` just for the sake of consistency?

Reply:
we already filter `@import` token and all we have is `"some import"` string and we cant put that match in first regex because it will brake it. I'll try to remove duplication though

Reply:
remove duplication. But its became more complex than it was